action.yml

```diff
@@ -87,280 +83,17 @@ inputs:
     description: 'HEAD SHA of a Pull Request. Used for posting statuses to the PR. The value is obtained from `git rev-parse HEAD` if this input is not set.'
     required: false
     default: ''
+  timeout:
+    description: 'Action is waiting for testing farm to finish or until timeout is reached.'
+    required: false
+    default: '480'
```

Comment:
Please write down, that 480 does it mean.

Reply:
I have added a note that the value is in minutes.